ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TipsLayout.kt

```diff
@@ -1,84 +1,164 @@
+                // when paused and play
+                // pass false to continue the animation
+                // at which is was paused
+                restartOnPlay = false
 
-            // this makes animation to restart
-            // when paused and play
-            // pass false to continue the animation
-            // at which is was paused
-            restartOnPlay = false
+            )
+            LottieAnimation(
+                composition,
+                { progress },
+                modifier = Modifier
+                    .height(290.dp)
+                    .fillMaxWidth()
+            )
 
-        )
-        LottieAnimation(
-            composition,
-            { progress },
-            modifier = Modifier.height(290.dp).fillMaxWidth()
-        )
+            Text(
+                text = stringResource(title),
+                style = MaterialTheme.typography.titleLarge,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.padding(horizontal = 32.dp)
+            )
 
-        Text(
-            text = stringResource(R.string.tutorial_tips_learn_headline),
-            style = MaterialTheme.typography.titleLarge,
-            textAlign = TextAlign.Center,
+            Text(
+                stringResource(body),
+                style = MaterialTheme.typography.bodyMedium,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.padding(horizontal = 32.dp)
+            )
 
-        )
+            if (body2 != null) {
+                Spacer(
+                    modifier = Modifier.minLinesHeight(
+                        minLines = 1,
+                        textStyle = MaterialTheme.typography.bodyMedium
+                    )
+                )
+                Text(
+                    stringResource(body2),
+                    style = MaterialTheme.typography.bodyMedium,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier.padding(horizontal = 32.dp)
+                )
+            }
+        }
+        Box(
+            modifier = Modifier.weight(1f)
+        ) {
+            if (page == Page.TIPS_START) {
+                Spacer(Modifier.padding(top = 150.dp))
+                Button(
+                    onClick = {
+                        onTutorialAction(R.id.action_tips_finish)
+                    },
 
-        Text(
-            stringResource(R.string.tutorial_tips_learn_text),
-            style = MaterialTheme.typography.bodyLarge,
-            textAlign = TextAlign.Center,
-            modifier = Modifier.padding(top = 16.dp)
-        )
+                    modifier = Modifier.padding(horizontal = 32.dp).width(275.dp)
+                ) {
+                    Text(
+                        text = stringResource(id = R.string.tutorial_tips_action_start),
+                        fontSize = 30.sp,
+                        modifier = Modifier.padding(horizontal = 32.dp)
```

Comment:
oops, I meant put the horizontal padding on all Text nodes that are all "root" elements, it doesn't make sense to put this padding on a Text node within the button.

The horizontal padding was meant to provide margin on both sides of the page. We didn't put it on the root layout node because we wanted the animation/image to be able to fill the entire page width.